### ISSUE TITLE
Logos not displaying properly

### ISSUE THREAD

**Issue by OliverEvans96** [Yash-Handa/logo-ls#4]
Hi,

Looks like a cool project, but it's not working for me. I'm mostly getting random Arabic and horoscope symbols displaying. I'm using gnome-terminal on Arch Linux with zsh (although it looks the same in bash). Let me know if I've misconfigured it somehow.

![2020-09-22-170248_1916x1041_scrot](https://user-images.githubusercontent.com/15915043/93937335-937f4500-fcf5-11ea-8cca-c2d97368c881.png)

Oliver

**Comment by OliverEvans96** [comment]
Related: https://www.reddit.com/r/golang/comments/ixrinq/vscode_like_icons_and_git_integration_in_terminal/g68jly8?utm_source=share&utm_medium=web2x&context=3

**Comment by mholt** [comment]
I'm getting the same thing as posted on ^ reddit (https://ibb.co/8NrQ6Ks) -- I imagine I'm missing some font or something? (Stock Ubuntu 20.04 terminal)

The screenshots in the readme have a much fancier shell than mine; will this not work with bash? A Requirements section would be helpful I think.

**Comment by Yash-Handa** [comment]
In order to get icons on the terminal [nerd fonts](https://github.com/ryanoasis/nerd-fonts) are required. Basically Nerd Fonts patches your current font i.e., the last few unicode points (approx 2,824 out of 143,859) in the font are replaced with nerdy icons.

The list of already patched fonts is given [here](https://github.com/ryanoasis/nerd-fonts#patched-fonts),  just choose a font from it, install it and change the terminal font to that font (restarting of terminal may be required).

You can also [patch a font](https://github.com/ryanoasis/nerd-fonts#option-8-patch-your-own-font) if you can't find your favorite font in the list.

-------

Some Platform dependent Instuctions:

- **Prefer not to use Nerd-font AUR**:
  Don't use _Nerd-fonts Complete_ AUR, it downloads all patched fonts available which is not what is required. You just need one font or you can patch your own font if you cant find it in the nerd-font repository.

  There are some single font Nerd-font AURs like for _DejaVo_, _Fira Code_, _Go Mone_ which can be used the list of available AUR packages is given [here](https://github.com/ryanoasis/nerd-fonts#option-7-unofficial-arch-user-repository-aur)

- **Using Homebrew Fonts on MacOS**:
  All fonts are available via [Homebrew Cask Fonts](https://github.com/Homebrew/homebrew-cask-fonts) on macOS (OS X). For more detail see [here](https://github.com/ryanoasis/nerd-fonts#option-4-homebrew-fonts)

**Comment by GiuBravo** [comment]
I'm facing the same issue on Ubuntu 22.04 LTS. I downloaded the fonts available on Nerdfonts, extracted them to the corresponding folder, and applied the font-patcher provided by Nerdfonts. I updated the cache, restarted the terminal, but it didn't resolve the issue.

**Comment by ISaIGoI** [comment]
Help!
 
![Screenshot 2023-11-21 191916](https://github.com/Yash-Handa/logo-ls/assets/25899258/2fe92678-a837-4550-a020-fe21c60bf5c0)
![Screenshot 2023-11-21 181448](https://github.com/Yash-Handa/logo-ls/assets/25899258/e9016ca2-e723-4477-af03-771427dd7505)

**Comment by LunaVaio** [comment]
I have the same issue. Using Ubuntu on WSL:
![image](https://github.com/Yash-Handa/logo-ls/assets/17891812/8a4bf968-5e78-4f10-a82e-d47e5b8c520d)

I've already installed the patched version of CascadiaCode and installed it, selected it in the settings:
![image](https://github.com/Yash-Handa/logo-ls/assets/17891812/774ac78a-62c2-44d7-8d48-18a361055fe5)

but still seeing the same result :(

**Comment by ISaIGoI** [comment]
Same thing!

**Comment by HamammSoleman** [comment]
I Also have the same issue, I'm currently using Cascadia Code.
Seems like it could be something to do with Ubuntu on wsl.

**Comment by RohollahHS** [comment]
I have the same issue.

**Comment by ISaIGoI** [comment]
> I Also have the same issue, I'm currently using Cascadia Code. Seems like it could be something to do with Ubuntu on wsl.

Nope, I did the same steps on my Linux VM, and the issue still exists!

**Comment by gabordikan** [comment]
I have the same issue under Windows. In Windows Terminal, I tried many of Nerd Fonts, but icons look same
![image](https://github.com/Yash-Handa/logo-ls/assets/77804349/d9e49f8d-9d6a-4b89-a265-72f5e948b91c)
![image](https://github.com/Yash-Handa/logo-ls/assets/77804349/245a55ed-bbcc-4e58-b558-80ed74ba5ea3)

**Comment by gabordikan** [comment]
Ok, this code fix has resolves my issue:
https://github.com/Yash-Handa/logo-ls/issues/49

**Comment by Abdelaziz-Elshrkawy** [comment]
hey guys i had this problem ton time and after a 1000 years 😁 of search found this packed font file that work with logo-ls so i saved it in my drive [font link](https://drive.google.com/file/d/1CMQjSQL5QZXRwh34pViGReramwDkTEIT/view?pli=1)

used it in Ubuntu wsl 

<img width="264" height="1560" alt="Image" src="https://github.com/user-attachments/assets/47320048-617c-4645-ab92-80ca41adcc2a" />


i hope it works

**Comment by Abdelaziz-Elshrkawy** [comment]
another thing is that some font files just not have the mentioned glyphs here 
[iconsMap.go](https://github.com/Yash-Handa/logo-ls/blob/master/assets/iconsMap.go)

so it just doesn't work so you can download source code change each one to your glyph code in the font you use and use  [fontforge](https://fontforge.org/en-US/) to preview the font glyph and code or just tell gpt to edit the map but any way not sure if it can work

**Comment by Abdelaziz-Elshrkawy** [comment]
another way to make it work found a repo to another [logo-ls](https://github.com/canta2899/logo-ls)

it work without any issues and even with bigger icons i think it can work on Windows and Linux
